front/src/modules/ui/components/buttons/Button.tsx

```diff
@@ -130,23 +147,47 @@ const StyledButton = styled.button<
   }
 `;
 
+const SoonPill = styled.span`
+  align-items: center;
+  background: ${({ theme }) => theme.background.transparent.light};
+  border-radius: 50px;
+  color: ${({ theme }) => theme.font.color.light};
+  display: flex;
+  font-family: Inter;
```

Comment:
@FelixMalfait Not sure if we need that, but if it's needed the constant is available in the theme:
```css
font-family: ${({ theme }) => theme.font.family};
```